app/src/main/res/layout/activity_connexion.xml

```diff
@@ -15,7 +14,7 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/a_connexion_text_view_app_name"
-        app:srcCompat="@drawable/ic_comif_logo" />
```

Comment:
Why?

Reply:
Fixed. Wanted to test it without the logo for UI tests, and forgot to re-add it after. The design remains more refined without the icon in my opinion !

Reply:
Deleted this line to do some tests on the UI. I re-added it, as I forgot to put it back. Yet I still think the design is more refined without the logo on both connexion and first connexion activities